build.sbt

```diff
@@ -109,7 +109,12 @@ lazy val core = crossProject(JVMPlatform)
     Compile / doc / scalacOptions ++= Seq(
       "-no-link-warnings" // Suppresses problems with Scaladoc @throws links
     ),
-    mimaBinaryIssueFilters := Nil
+    mimaBinaryIssueFilters := List(
```

Comment:
when bumping series above this is not needed.

Reply:
Ah, makes sense. I've removed them.